lua/nvim-tree/actions/fs/rename-file.lua

```diff
@@ -24,25 +30,54 @@ function M.rename(node, to)
   events._dispatch_node_renamed(node.absolute_path, to)
 end
 
-function M.fn(with_sub)
-  return function(node)
+function M.fn(default_modifier)
+  default_modifier = default_modifier or ":t"
+
+  return function(node, modifier)
```

Comment:
I removed capability for the first argument to be a modifier after adding the API, as I couldn't see a use case.

I may be missing something...